aptos-move/framework/aptos-framework/sources/dora_committee.move

```diff
@@ -0,0 +1,1042 @@
+/// Design:
+/// CommitteeInfoStore: store all the committee information, supported operation: add, remove
+/// CommitteeInfo: store all the dora node information, supported operation: add, remove
+/// DoraNodeInfo: store the dora node information, supported operation: update
+/// NodeData: store the dora node information with operator's address
+/// requirements:
+/// 1. two committee has different id
+/// 2. two indentical node should not be in the same committee
+/// 3. the operator
+module aptos_framework::dora_committee {
+    use std::error;
+    use std::signer;
+    use std::vector;
+    use aptos_std::capability;
+    use aptos_std::simple_map;
+    use aptos_std::simple_map::SimpleMap;
+    use aptos_framework::account;
+    use aptos_framework::account::new_event_handle;
+    use aptos_framework::event::{emit_event, EventHandle};
+
+    /// The signer is not the admin of the committee contract.
+    const ENOT_ADMIN: u64 = 1;
```

Comment:
I can't find `ENOT_ADMIN` variable being used anywhere within the contract. If it's unused, we can remove it

Reply:
@nizam-supraoracles Thanks, addressed in new commit.